frontend/src/app/shared/components/team-members-window/team-members-window.component.ts

```diff
@@ -0,0 +1,102 @@
+import { Component, ElementRef, EventEmitter, Inject } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { BaseComponent } from '@core/base/base.component';
+import { ITeamMember } from '@core/models/ITeamMember';
+import { IUser } from '@core/models/IUser';
+import { NotificationService } from '@core/services/notification.service';
+import { TeamService } from '@core/services/team.service';
+import { UserService } from '@core/services/user.service';
+import { Role } from '@shared/enums/role';
+import { Status } from '@shared/enums/status';
+import { IConfirmButtonOptions } from '@shared/models/confirmWindow/IConfirmButtonOptions';
+import { ITeamMembersDialogData } from '@shared/models/ITeamMembersDialogData';
+
+@Component({
+    selector: 'app-team-members-window',
+    templateUrl: './team-members-window.component.html',
+    styleUrls: ['./team-members-window.component.sass'],
+})
+export class TeamMembersWindowComponent extends BaseComponent {
+    title: string;
+
+    message?: string;
+
+    buttonsOptions?: IConfirmButtonOptions[];
+
+    teamMembers?: ITeamMember[];
+
+    teamId?: number;
+
+    searchedUsers: IUser[];
+
+    query: string;
+
+    usersToAdd: IUser[] = [];
+
+    constructor(
+        @Inject(MAT_DIALOG_DATA) public data: ITeamMembersDialogData,
+        private dialogRef: MatDialogRef<TeamMembersWindowComponent>,
+        private userService: UserService,
+        private teamService: TeamService,
+        private el: ElementRef,
+        private notificationService: NotificationService,
+    ) {
+        super();
+        this.title = data.title;
+        this.buttonsOptions = data.buttonsOptions;
+        this.teamMembers = data.teamMembers;
+        this.teamId = data.teamId;
+        this.message = data.message;
+    }
+
+    onClick(event: EventEmitter<void>) {
+        event?.next();
+        this.dialogRef.close();
+    }
+
+    getUsersByEmailOrName(searchData: string) {
+        if (searchData.length !== 0) {
+            this.userService
+                .getUsersByEmailOrName(searchData)
+                .pipe(this.untilThis)
+                .subscribe(
+                    (users) => {
+                        this.searchedUsers = users;
+                    },
+                    (error) => {
+                        this.notificationService.showErrorMessage(error);
+                    },
+                );
+        } else {
+            this.searchedUsers = [];
+        }
+    }
+
+    isAddedMember(user: IUser) {
+        return this.teamMembers?.some((p) => p.email === user.email);
+    }
+
+    selectUser(user: IUser) {
+        const element: HTMLDivElement = this.el.nativeElement.querySelector(`#user-info-${user.id}`) as HTMLDivElement;
+
+        element.classList.add('window-selected-user');
+        element.classList.remove('not-added-user-info');
```

Comment:
Use conditional class instead

Reply:
Do you mean this?
```suggestion
        this.el.nativeElement.querySelector(`#user-info-${user.id}`).classList.add('window-selected-user');
        this.el.nativeElement.querySelector(`#user-info-${user.id}`).classList.remove('not-added-user-info');
```

Reply:
I mean can u don't use nativeElement, and use https://angular.io/api/common/NgClass instead?

Reply:
I am not sure It`s possible to implement this with NgClass.
I have to change style of each div with selected user and do it by id:
![image](https://user-images.githubusercontent.com/71783287/189346396-51bd7e8e-52bd-436b-a8d0-13fa2fcbf12f.png)

Reply:
Its possible. Add method that will check if userId is selected and call this from ngClass